src/errorpage.cc

```diff
@@ -1203,6 +1197,7 @@ ErrorState::compileLegacyCode(Build &build)
         if (Config.adminEmail && Config.onoff.emailErrData)
             Dump(&mb);
         no_urlescape = 1;
+        do_quote = 0;
```

Comment:
Is this the bug fix? Please disclose the fixed bug in the PR description.

Reply:
done